packages/core/src/date-utils.js

```diff
@@ -68,23 +68,23 @@ function toDate(input) {
 }
 
 function formatAsDate(obj) {
-    return obj.format(DB_DATE_FORMAT);
+    return dayjs(obj).format(DB_DATE_FORMAT);
```

Comment:
Made these changes so you can use dateUtils.formatAsDate() and get the current date formatted in that format